### PR TITLE
Important information that isn't mentioned

### PR DESCRIPTION
**Pull request by haflidif**
Hi,
I recently found out if you are trying to pre-provision OneDrive for business accounts, it will fail if the user account is blocked for sign-in (Disabled), so I added a Note that states that all effective users or those user/users that need their ODFB pre-provisioned needs to be allowed for sign-in.

Can you please add this to the documentation.
Kind Regards
Haflidi Fridthjofsson.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/pre-provision-accounts.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ This article describes how to pre-provision OneDrive for your users by using Pow
     > If you installed a previous version of the SharePoint Online Management Shell, go to Add or remove programs and uninstall “SharePoint Online Management Shell.” <br>On the Download Center page, select your language and then click the Download button. You’ll be asked to choose between downloading a x64 and x86 .msi file. Download the x64 file if you’re running the 64-bit version of Windows or the x86 file if you’re running the 32-bit version. If you don’t know, see https://support.microsoft.com/help/13443/windows-which-operating-system. After the file downloads, run it and follow the steps in the Setup Wizard.
     
 3. Connect to SharePoint Online as a global admin or SharePoint admin in Office 365. To learn how, see [Getting started with SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online).
+
+    > [!NOTE]
+    > To be able to run the Pre-provision PowerShell command all effective users needs to be allowed for sign-in.
```
Comment (JohanFreelancer9) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
    > To be able to run the Pre-provision PowerShell command, all effective users need to be allowed for sign-in.
```

**Comment by haflidif** [comment]
I've commited the changes from @JohanFreelancer9  👍 thank you

**Comment by scanum** [comment]
@haflidif PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: OneDrive/pre-provision-accounts.md [hunk 2]
```diff
@@ -50,6 +50,9 @@ This article describes how to pre-provision OneDrive for your users by using Pow
     > If you installed a previous version of the SharePoint Online Management Shell, go to Add or remove programs and uninstall “SharePoint Online Management Shell.” <br>On the Download Center page, select your language and then click the Download button. You’ll be asked to choose between downloading a x64 and x86 .msi file. Download the x64 file if you’re running the 64-bit version of Windows or the x86 file if you’re running the 32-bit version. If you don’t know, see https://support.microsoft.com/help/13443/windows-which-operating-system. After the file downloads, run it and follow the steps in the Setup Wizard.
     
 3. Connect to SharePoint Online as a global admin or SharePoint admin in Office 365. To learn how, see [Getting started with SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online).
+
```
Comment (nenonix) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
```

File: OneDrive/pre-provision-accounts.md [hunk 3]
```diff
@@ -50,6 +50,9 @@ This article describes how to pre-provision OneDrive for your users by using Pow
...
+    > [!NOTE]
```
Comment (nenonix) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
```

File: OneDrive/pre-provision-accounts.md [hunk 4]
```diff
@@ -50,6 +50,9 @@ This article describes how to pre-provision OneDrive for your users by using Pow
...
+    > To be able to run the Pre-provision PowerShell command, all effective users need to be allowed for sign-in.
```
Comment (nenonix) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
```

File: OneDrive/pre-provision-accounts.md [hunk 5]
```diff
@@ -50,6 +50,9 @@ This article describes how to pre-provision OneDrive for your users by using Pow
...
     
 4. Run the PowerShell command [Request-SPOPersonalSite](/powershell/module/sharepoint-online/request-spopersonalsite?view=sharepoint-ps), consuming the text file you previously created in Step 1.
```
Comment (nenonix) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
>  [!IMPORTANT]
> To run the **Request-SPOPersonalSite** PowerShell command, you need to allow all effective users for sign-in.
    	    
```

**Comment by haflidif** [comment]
I think my pull request should be ready for merging now, as I've added all suggested updates, and include little additional information.

File: OneDrive/pre-provision-accounts.md [hunk 6]
```diff
@@ -50,18 +50,28 @@ This article describes how to pre-provision OneDrive for your users by using Pow
     > If you installed a previous version of the SharePoint Online Management Shell, go to Add or remove programs and uninstall “SharePoint Online Management Shell.” <br>On the Download Center page, select your language and then click the Download button. You’ll be asked to choose between downloading a x64 and x86 .msi file. Download the x64 file if you’re running the 64-bit version of Windows or the x86 file if you’re running the 32-bit version. If you don’t know, see https://support.microsoft.com/help/13443/windows-which-operating-system. After the file downloads, run it and follow the steps in the Setup Wizard.
     
 3. Connect to SharePoint Online as a global admin or SharePoint admin in Office 365. To learn how, see [Getting started with SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online).
+
+    > [!IMPORTANT]
+    > To run the **Request-SPOPersonalSite** PowerShell command, you need to allow all effective users for sign-in.
     
+    > [!NOTE]
+    > If you need to allow effective users for sign-in you can run the Powershell command **Set-MsolUser**, you can consume the text file you previously created in Step 1.
```
Comment (JohanFreelancer9) [inline OneDrive/pre-provision-accounts.md]:
```suggestion
    > If you need to allow effective users for sign-in, you can run the Powershell command **Set-MsolUser**, and you can consume the text file you previously created in Step 1.
```

**Comment by haflidif** [comment]
@kaarins and @scanum I guess everything is in place now, and @JohanFreelancer9 has copy reviewed the PR, can it be merged and closed now, or do you have more suggestions ?

**Pull request merged**